katrain/i18n/locales/ua/LC_MESSAGES/katrain.po

```diff
@@ -185,8 +178,8 @@ msgstr "Визначити область для аналізу"
 #. Status message
 msgid "analysis:region:start"
 msgstr ""
-"Потягніть для вибору області інтересу. Виберіть одну точку для"
-"скасування області."
+"Потягніть для вибору області інтересу. Виберіть одну точку для скасування "
+"області."
```

Comment:
In the past PR #712 I formatted this file manually rather than used the i18n.py script. So there would be many unrelated changes in this file.